UnityProject/Assets/Scripts/Core/UniversalObjectPhysics.cs

```diff
@@ -1269,7 +1271,29 @@ public void SetRegisterTileLocation(Vector3Int localPosition)
 			maxDistanceDelta);
 	}
 
-	[PlayModeOnly] public bool IsFlyingSliding = false; //Is animating with space flying
+	[PlayModeOnly] private float SecondsFlying;
+
+	public bool IsFlyingSliding
+	{
+		get
+		{
+			return false;
```

Comment:
Just a little confused on why we always return false - what's the point of this then?